docs/source/sections/Installation.rst

```diff
@@ -10,34 +10,93 @@ Windows
 #######
 The PolarRoute software requires GDAL files to be installed. The PolarRoute software can be installed on Windows by running one of the two following commands.
 
-Github:
+.. note:: 
+    We assume a version of Windows 10 or higher, with a working version of Python 3.9 including pip installed. 
+    We recommend installing PolarRoute into a virtual environment.
+
+Requirements:
 ::
```

Comment:
@foxm1 
2. There is an issue around this currently set to be developed as part of the 1.1 release of the Mesh Builder https://github.com/antarctica/SDADT-project/issues/150. Would you like this moved to the 0.2.x release of PolarRoute instead?

3. There is an issue around this current set to be developed as part of the 1.1 release of the Mesh Builder https://github.com/antarctica/SDADT-project/issues/157. Would you like this moved to the 0.2.x release of PolarRoute?

4. This will be addressed as part of the 0.3.x release of PolarRoute.

7. A `ub%: 1` and` lb%:  0` would not result in a cellbox always being split, this would result in a cellbox being split in _all cases except_ when **all** datapoints in the cellbox are either above or below a given threshold value.